<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(121, 121, 121);</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>222</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_adres"/>
    </item>
    <item row="6" column="4">
     <widget class="QLineEdit" name="lineEdit_ehliyet"/>
    </item>
    <item row="6" column="3">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ehliyet Sınıfı</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Soyad</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="lineEdit_telno"/>
    </item>
    <item row="7" column="3">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Medeni Durum</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QLineEdit" name="lineEdit_egitim"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Doğum Tarihi</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ad</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Adres</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_ad"/>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Meslek</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tc Kimlik NO</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QLineEdit" name="lineEdit_medeni"/>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tel No</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_tcno"/>
    </item>
    <item row="5" column="4">
     <widget class="QLineEdit" name="lineEdit_meslek"/>
    </item>
    <item row="8" column="3">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Eğitim Durumu</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="lineEdit_soyad"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_dTarihi"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>User ID</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QPushButton" name="pushButton_sil">
      <property name="text">
       <string>Sil</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QPushButton" name="pushButton_kaydet">
      <property name="text">
       <string>Kaydet</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="pushButton_update">
      <property name="text">
       <string>Güncelle</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="pushButton_yeni">
      <property name="text">
       <string>Yeni</string>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QPushButton" name="pushButton_Getir2">
      <property name="text">
       <string>Getir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>1091</width>
     <height>431</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>4</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>5</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>6</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>7</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>8</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>9</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>10</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>11</string>
    </property>
   </column>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>lineEdit_soyad</tabstop>
  <tabstop>lineEdit_ad</tabstop>
  <tabstop>lineEdit_telno</tabstop>
  <tabstop>lineEdit_tcno</tabstop>
  <tabstop>lineEdit_meslek</tabstop>
  <tabstop>lineEdit_dTarihi</tabstop>
  <tabstop>lineEdit_ehliyet</tabstop>
  <tabstop>lineEdit_adres</tabstop>
  <tabstop>lineEdit_medeni</tabstop>
  <tabstop>lineEdit_egitim</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>pushButton_sil</tabstop>
  <tabstop>pushButton_kaydet</tabstop>
  <tabstop>pushButton_update</tabstop>
  <tabstop>pushButton_yeni</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
